src/Compilers/CSharp/Portable/Symbols/Source/SourceNamedTypeSymbol.cs

```diff
@@ -1812,6 +1815,16 @@ internal bool IsSimpleProgram
             }
         }
 
+        public override string MetadataName
+        {
+            get
+            {
+                return IsExtension
+                    ? MetadataHelpers.ComposeAritySuffixedMetadataName(Name, Arity, this.ExtensionName)
```

Comment:
>Name [](http://example.com/codeflow?start=71&length=4)

What is the ```Name``` value here?

Reply:
It's blank, but I should be using `ExtensionName` here instead of `Name`. Thanks